src/design-system/Modal.tsx

```diff
@@ -12,10 +12,12 @@ export interface Props {
   open: boolean;
   setOpen: (open: boolean) => void;
   trigger?: string | React.ReactElement<any>;
+  height?: string;
+  width?: string;
```

Comment:
With these new properties in place we can size our modals to meet the needs of a given context.  I've done this to support the Scenario Library, but we can also use this to resize our current confirmation modals which are a bit awkward in size using the defaults